### ISSUE TITLE
Allow a 'vertical' tree style when plotting trees?

### ISSUE THREAD

**Issue by ndphillips** [ndphillips/FFTrees#222]
It has occurred to me that the way FFTs are visualized both in this package as well as in publications is in what I call a 'wide' orientation. In an 'width' orientation, nodes of the tree are arranged horizontally based on the exit structure of the preceding node. That is, a node below a 'left-exit' node will be placed to the right of the node above it, while a node below a 'right-exit' node will be placed to the left of the node above it.

By contrast, consider what I would call a 'vertical' visualization, where nodes are always shown vertically stacked on top of each other, regardless of the node exit.

See my crude drawing (not to scale!) below

<img width="608" alt="image" src="https://github.com/ndphillips/FFTrees/assets/8480980/86d1becd-9e93-40fa-ad6c-e19233d606e7">

I can think of two benefits to plotting trees using this vertical orientation within `FFTrees()`:

1. Vertical orientations are much simpler to construct and will reduce the amount of code (there's a bunch of code in https://github.com/ndphillips/FFTrees/blob/master/R/plotFFTrees_function.R that figures out where elements should be placed based on the location of the prior node. If we used a vertical orientation, we could remove all of that logic.
2. Vertical orientations will lead to more predictable widths of trees and more consistent visualization dimensions. If I was embedding plots of FFTrees in a dashboard or paper, I'd appreciate this!
3. Vertical orientations emphasize the simplicity of ffts. That is, unlike information in 'standard' trees, information in ffts flows in one direction only -- down. 

@hneth curious what you think about this. Is this something you've thought of or come across before?

**Comment by hneth** [comment]
Yes, I've also considered alternative tree layouts and also seen vertical designs in various publications.  And I agree that a vertical arrangement of nodes has its advantages (e.g., the layout gets simpler and less dependent on exit structures).  Aesthetically, vertical designs seem cleaner, but less recognizable than our current design.

Perhaps an idea for an even simpler tree layout would be to allow for placing exits horizontally next to the cue nodes (as shown in the sketch below).  If we were willing to relax the strict "noise = left" and "signal = right" convention, we could also create even narrower designs by arranging all exits on one side of the nodes (see 2nd sketch below).

[FFT_layouts.pdf](https://github.com/ndphillips/FFTrees/files/15414215/FFT_layouts.pdf)

But all these and potential future layouts could easily be created if we had a modular and well-designed `plot_FFT()` function. Given the importance of a good FFT visualization function for package users, that's clearly a worthwhile task to address.
